.circleci/config.yml

```diff
@@ -221,4 +236,9 @@ workflows:
                 - LOAD8
                 - LOAD9
                 - LOAD10
-
+  rootless:
```

Comment:
rootless snuck back in.  Maybe just comment it out?  It's gone on master.

Reply:
very sneaky. Will take it back out